### ISSUE TITLE
Some stdlib documentation is missing.

### ISSUE THREAD

**Issue by Aerilius** [Aerilius/sketchup-console-plus#3]
Some stdlib docs are missing from apis in DocProvider. Find out whether yardoc was not able to find or extract them from Ruby C source code or whether our `generate_ruby_apis.rb` excludes them from the generated apis.

## Test case:
```
AE::ConsolePlugin.DocProvider.get_infos_for_doc_path('FileUtils')
>> []
```
This should include doc info for `FileUtils.cp`

**Comment by MSP-Greg** [comment]
I don't think [generate_ruby_apis.rb:44 includes](https://github.com/Aerilius/sketchup-console-plus/blob/master/tools/generate_ruby_apis.rb#L44) is picking up `lib/*.rb`.

Some std-lib's have folders, but almost all have their base class in `lib`.

Also, as to extensions, you can probably just glob `ext/**/*.{rb,c}`...

**Comment by Aerilius** [comment]
Thanks, I'm looking into it!
I was careful including folders, because the first time parsing all the source with yardoc, it parsed many unimportant files, specs, tests.

**Comment by Aerilius** [comment]
This seems to be fixed with 8580bd3ed00f69885ca13f84be69ee03831eb3fb
